room_assistance/indico_room_assistance/migrations/20190220_1138_b160812c186b_add_startup_assistance_requests_table.py

```diff
@@ -0,0 +1,100 @@
+"""Add room assistance requests table
```

Comment:
iirc. we wanted to move this to a standalone script like for vc assistance

Reply:
while using the models in such a script would be cleaner, feel free to keep the SQL since that way it's minimal work to move it to the standalone script